totalRP3_Extended/aura/aura.xml

```diff
@@ -0,0 +1,244 @@
+					<Anchors>
+						<Anchor point="TOPLEFT"/>
+						<Anchor point="BOTTOMRIGHT" relativePoint="TOPRIGHT" x="0" y="-16"/>
+					</Anchors>
+					<Color r="0.9" g="0.9" b="0.9"/>
+				</FontString>
+			</Layer>
+			<Layer level="BACKGROUND">
+				<Texture parentKey="BottomLeft" file="Interface\FrameGeneral\UIFrameBackground" hidden="true">
+					<Size x="16" y="16"/>
+					<Anchors>
+						<Anchor point="BOTTOMLEFT"/>
+					</Anchors>
+					<TexCoords top="0.03125" left="0.015625" bottom="0.53125" right="0.265625"/>
+					<Color r="0.12" g="0.12" b="0.12" a="0.5"/>
+				</Texture>
+				<Texture parentKey="BottomRight" file="Interface\FrameGeneral\UIFrameBackground" hidden="true">
+					<Size x="16" y="16"/>
+					<Anchors>
+						<Anchor point="BOTTOMRIGHT"/>
+					</Anchors>
+					<TexCoords top="0.03125" left="0.265625" bottom="0.53125" right="0.015625"/>
+					<Color r="0.12" g="0.12" b="0.12" a="0.5"/>
+				</Texture>
+				<Texture parentKey="TopLeft" file="Interface\FrameGeneral\UIFrameBackground" hidden="true">
+					<Size x="16" y="16"/>
+					<Anchors>
+						<Anchor point="TOPLEFT"/>
+					</Anchors>
+					<TexCoords top="0.53125" left="0.015625" bottom="0.03125" right="0.265625"/>
+					<Color r="0.12" g="0.12" b="0.12" a="0.5"/>
+				</Texture>
+				<Texture parentKey="TopRight" file="Interface\FrameGeneral\UIFrameBackground" hidden="true">
+					<Size x="16" y="16"/>
+					<Anchors>
+						<Anchor point="TOPRIGHT"/>
+					</Anchors>
+					<TexCoords top="0.53125" left="0.265625" bottom="0.03125" right="0.015625"/>
+					<Color r="0.12" g="0.12" b="0.12" a="0.5"/>
+				</Texture>
+				<Texture parentKey="TopEdge" hidden="true">
+					<Anchors>
+						<Anchor point="TOPLEFT" relativeKey="$parent.TopLeft" relativePoint="TOPRIGHT"/>
+						<Anchor point="BOTTOMRIGHT" relativeKey="$parent.TopRight" relativePoint="BOTTOMLEFT"/>
+					</Anchors>
+					<Color r="0.12" g="0.12" b="0.12" a="0.5"/>
+				</Texture>
+				<Texture parentKey="BottomEdge" hidden="true">
+					<Anchors>
+						<Anchor point="TOPLEFT" relativeKey="$parent.BottomLeft" relativePoint="TOPRIGHT"/>
+						<Anchor point="BOTTOMRIGHT" relativeKey="$parent.BottomRight" relativePoint="BOTTOMLEFT"/>
+					</Anchors>
+					<Color r="0.12" g="0.12" b="0.12" a="0.5"/>
+				</Texture>
+				<Texture parentKey="Center" hidden="true">
+					<Anchors>
+						<Anchor point="TOPLEFT" relativeKey="$parent.TopLeft" relativePoint="BOTTOMLEFT"/>
+						<Anchor point="BOTTOMRIGHT" relativeKey="$parent.BottomRight" relativePoint="TOPRIGHT"/>
+					</Anchors>
+					<Color r="0.12" g="0.12" b="0.12" a="0.5"/>
+				</Texture>
+			</Layer>
+		</Layers>
+		<Scripts>
+			<OnEnter>
+				self.Text:Show();
+				self.BottomLeft:Show();
+				self.BottomRight:Show();
+				self.TopLeft:Show();
+				self.TopRight:Show();
+				self.TopEdge:Show();
+				self.BottomEdge:Show();
+				self.Center:Show();
```

Comment:
More maintainable approach - add an `parentArray="mouseInteractiveRegions"` attribute to the Text, BottomLeft, etc. regions and then change the contents of these OnEnter/OnLeave handlers to loop over that list:

```lua
for _, region in ipairs(self.mouseInteractiveRegions) do
    region:Show();
end
```

(And move this to the mixin in a Lua script as mentioned in the other review comment)